### PR TITLE
refactor: update stack with component-rollup-config

### PR DESCRIPTION
**Pull request by cbourgois**
### Description of the Change

replace module name from 'ovh-ng-input-password' to 'ngOvhInputPassword'
BREAKING CHANGE: module is now named `ngOvhInputPassword`

f45c132 — refactor: update stack with component-rollup-config

ref: 

### TODO : 

* [x] rename repository to `ng-ovh-input-password`

/cc @jleveugle @frenautvh @antleblanc

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -1,16 +1,18 @@
...
-    "module": true,
-    "__dirname": true,
-    "inject": true
+  "rules": {
+    "import/no-unresolved": 0
   },
-  "env": {
-    "jasmine": true
-  }
+  "overrides": [{
```
Comment (antleblanc) [inline .eslintrc.json]: According to the `README.md` file, these `overrides` aren't needed.

File: .gitignore [hunk 2]
```diff
@@ -1,5 +1,2 @@
-node_modules/
-bower_components/
-.work
-.tmp
-src/translations/*.json
+node_modules
+dist
```
Comment (antleblanc) [inline .gitignore:2]: +`tools`

File: src/index.less [hunk 3]
```diff
@@ -0,0 +1,237 @@
+@oui-path: "../../../ovh-ui-kit";
+@import "~ovh-ui-kit-bs/src/bootstrap/_variables";
+
+@ovhNgInputPasswordPopoverBgColor: #f3f3f3;
```
Comment (antleblanc) [inline src/index.less]: Shouldn't be `kebabCase` instead?

**Review by antleblanc — CHANGES_REQUESTED** [review]

File: src/index.less (hunk 3 above)
Comment (cbourgois) [reply src/index.less]: Done :)

File: .lintstagedrc.json [hunk 4]
```diff
@@ -0,0 +1,6 @@
+{
+  "*.js": [
```
Comment (antleblanc) [inline .lintstagedrc.json:5]: Missing `*.less`.

File: package.json [hunk 5]
```diff
@@ -1,50 +1,57 @@
...
+  "main": "./dist/cjs/index.js",
+  "module": "./dist/esm/index.js",
+  "browser": "./dist/umd/ng-ovh-input-password.js",
   "scripts": {
-    "fix": "eslint --quiet --fix --ext .js src",
-    "test": "grunt test"
+    "build": "rollup -c",
+    "dev": "rollup -c --watch",
+    "lint": "run-p lint:js lint:md",
+    "lint:js": "eslint --quiet --fix --format=pretty src",
```
Comment (antleblanc) [inline package.json:26]: Missing `lint:css` as well.

File: package.json [hunk 6]
```diff
@@ -1,50 +1,57 @@
...
-    "karma": "~1.6.0",
-    "karma-jasmine": "~0.3.0",
-    "karma-ng-html2js-preprocessor": "^1.0.0",
-    "karma-phantomjs-launcher": "^1.0.1",
-    "karma-spec-reporter": "0.0.26",
-    "matchdep": "^0.3.0",
-    "phantomjs-prebuilt": "^2.1.11"
+    "@commitlint/cli": "^7.1.2",
+    "@commitlint/config-angular": "^7.1.2",
+    "@ovh-ux/component-rollup-config": "5.0.0-beta.5",
```
Comment (antleblanc) [inline package.json]:
```suggestion
    "@ovh-ux/component-rollup-config": "5.0.0-beta.6",
```

File: src/controller.js [hunk 7]
```diff
@@ -0,0 +1,125 @@
+import forEach from 'lodash/forEach';
+import isFunction from 'lodash/isFunction';
+import isRegExp from 'lodash/isRegExp';
+import kebabCase from 'lodash/kebabCase';
+
+
```
Comment (antleblanc) [inline src/controller.js]: extra empty line.

File: src/translations/Messages_fr_FR.xml [hunk 8]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (antleblanc) [inline src/translations/Messages_fr_FR.xml:1]: Could we keep only XML file?

File: src/toggle/index.js [hunk 9]
```diff
@@ -0,0 +1,16 @@
+import angular from 'angular';
+import 'angular-translate';
+
+import directive from './directive';
+
+// import './index.less';
```
Comment (antleblanc) [inline src/toggle/index.js]: Needed?

**Review by antleblanc — CHANGES_REQUESTED** [review]

File: package.json (hunk 6 above)
Comment (cbourgois) [reply package.json]: Bumped.

```suggestion
    "@ovh-ux/component-rollup-config": "5.0.0-beta.8",
```

File: package.json (hunk 5 above)
Comment (cbourgois) [reply package.json:26]: added `lint:html` too.

File: .lintstagedrc.json (hunk 4 above)
Comment (cbourgois) [reply .lintstagedrc.json:5]: `*.html` added too

File: src/translations/Messages_fr_FR.xml (hunk 8 above)
Comment (cbourgois) [reply src/translations/Messages_fr_FR.xml:1]: `json` files removed (and empty `en_GB` xml too)

**Review by antleblanc — APPROVED** [review]

**Pull request merged**